lib/float/common.sail

```diff
@@ -89,4 +89,46 @@ function float_decompose(op) = {
 val      not : forall ('p : Bool). bool('p) -> bool(not('p))
 function not(b) = not_bool(b)
 
+val      is_lowest_one : forall 'n, 0 < 'n. bits('n) -> bool
```

Comment:
Hi @Alasdair, I may consider moving these helper functions to the vector.sail, because they are not only for the floating lib. But I would like to learn some suggestions about it from the expert like you. 

Thanks a lot.

Reply:
Kindly reminder for the best reasonable place for these helper functions.

Reply:
I think we can put them in the float common file for now, then consider moving some later.

Reply:
Sure, thanks for confirmation.